x-pack/plugins/observability_solution/apm/public/components/routing/templates/custom_no_data_template.tsx

```diff
@@ -0,0 +1,122 @@
+  isPageDataLoaded,
+  noDataConfig,
+}: {
+  isPageDataLoaded: boolean;
+  noDataConfig?: NoDataConfig;
+}) {
+  const { services } = useKibana<ApmPluginStartDeps>();
+  const { http, observabilityShared } = services;
+  const basePath = http?.basePath.get();
+
+  const ObservabilityPageTemplate = observabilityShared.navigation.PageTemplate;
+  const imageUrl = `${basePath}/plugins/kibanaReact/assets/elastic_agent_card.svg`;
+
+  return (
+    <ObservabilityPageTemplate isPageDataLoaded={isPageDataLoaded} paddingSize="none">
+      <EuiPageTemplate panelled={false} offset={0} restrictWidth="960px">
+        <EuiPageTemplate.Section alignment="center" component="div" grow>
+          <EuiText textAlign="center">
+            <KibanaSolutionAvatar name="observability" iconType="logoObservability" size="xxl" />
+            <EuiSpacer size="l" />
+            <h1>
+              {i18n.translate('xpack.apm.customEmtpyState.title', {
+                defaultMessage: 'Detect and resolve problems with your application',
+              })}
+            </h1>
+            <EuiTextColor color="subdued">
+              <p>
+                {i18n.translate('xpack.apm.customEmtpyState.description', {
+                  defaultMessage:
+                    'Start collecting data for your applications and services so you can detect and resolve problems faster.',
+                })}
+              </p>
+            </EuiTextColor>
+          </EuiText>
+          <EuiSpacer size="xxl" />
+          <EuiCard
+            css={{ maxWidth: 400, marginInline: 'auto' }}
+            paddingSize="l"
+            title={
+              <EuiScreenReaderOnly>
+                <span>
+                  {i18n.translate('xpack.apm.customEmtpyState.title.reader', {
+                    defaultMessage: 'Add APM data',
+                  })}
+                </span>
+              </EuiScreenReaderOnly>
+            }
+            description={i18n.translate('xpack.apm.customEmtpyState.card.description', {
+              defaultMessage:
+                'Use APM agents to collect APM data. We make it easy with agents for many popular languages.',
+            })}
+            footer={
+              <EuiFlexGroup justifyContent="center">
+                <EuiFlexItem grow={false}>
+                  <EuiButton
+                    data-test-subj="apmCustomNoDataTemplateAddApmAgentButton"
+                    color="primary"
+                    fill
+                    href={noDataConfig?.action.elasticAgent.href}
+                  >
+                    {noDataConfig?.action.elasticAgent.title}
+                  </EuiButton>
+                  <EuiSpacer size="m" />
+                  <EuiText size="s">
+                    <p>
+                      <EntityEnablement
+                        label={i18n.translate('xpack.apm.customEmtpyState.card.link', {
+                          defaultMessage: 'Try creating services from logs',
```

Comment:
Is `creating` the right wording here? I wonder if `collecting` would sound better.

Reply:
CC @roshan-elastic

Reply:
Yeah good shout @crespocarlos @kpatticha. Let's use 'collecting'

Reply:
updated it [here](https://github.com/elastic/kibana/pull/188918/commits/c84a145ad8d08fe786cc2aa2d36eabc72e3b0bfc) since this PR is already merged